constrained-voucher.mkd

```diff
@@ -313,20 +313,38 @@ For example, if the Registrar supports a short BRSKI URL (/b) and supports the v
 The Registrar is under no obligation to provide shorter URLs, and MAY respond to this query with only the "/.well-known/brski/\<short-name\>" end points for the short names as defined in {{est-uri}}.
 
 Registrars that have implemented shorter URLs MUST also respond in equivalent ways to the corresponding "/.well-known/brski/\<short-name\>" URLs, and MUST NOT distinguish between them.
-In particular, a Pledge MAY use the longer and shorter URLs in combination.
+In particular, a Pledge MAY use the longer and shorter URLs in any combination.
 
 When responding to a discovery request for BRSKI resources, the server MAY in addition return
 the full resource paths and the content types which are supported at those end-points as shown in above example.
 This is useful when multiple content types are specified for a particular resource on the server.
 The server responds with only the root path for the BRSKI resource (rt=brski, resource /b in above example) and no others in case the client queries for only rt=brski type resources.
 (So, a query for rt=brski, without the wildcard character.)
 
+Without the shorter paths, a full URL is used, such as:
```

Comment:
Without discovery, a longer well-known URL can only be used, such as:

Reply:
I would not say "without the shorter paths" which could be confused with "shorter path names" in earlier paragraphs. The latter is referring to the "<short-name>" resources which is still used in the example below GET /.well-known/brski/rv  . So that sounds like a contradiction.   

Maybe we need different naming to denote the "path" and the "resource name" of the final resource.